betterforms/tests.py

```diff
@@ -332,6 +333,7 @@ class Meta:
                 )
         self.TestForm = TestForm
 
+    @unittest.skipIf(django.VERSION < (1, 8), 'HTML output is slightly different in 1.8')
```

Comment:
Instead of just skipping these tests on 1.7, what if you did something like:

```python
test = """...."""
if django.VERSION < (1,8):
    test = test.replace('class="required" ', '')
```

I'm just very uncomfortable not running the tests on 1.7